features/carrinho_compras.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Carrinho de compras
+As a usuário
+I want escolher os itens que desejo comprar
+So that eu possa efetuar o pagamento e receber meus produtos
+
+Scenario: Carrinho de compras vazio
+Given que o usuário com CPF "00000000001" está na página "carrinho_de_compras"
+And o carrinho de compras está vazio
+When o usuário de CPF "00000000001" acessa a página "carrinho_de_compras"
+Then a mensagem "Seu carrinho está vazio." é exibida
+And o usuário pode "Começar a comprar"
+And a opção "Finalizar pedido" não é exibida na página
+
+Scenario: Carrinho de compras abandonado
+Given que o usuário com CPF "00000000001" está na página "principal"
+And o usuário de CPF "00000000001" adiciona os itens "Camisa Puma - masculino" e "Camisa Adidas - masculino" ao carrinho de compras
+When o usuário de CPF "00000000001" sai do site
+And depois volta ao site
+Then os itens "Camisa Puma - masculino" e "Camisa Adidas - masculino" continuam presentes no carrinho de compras
+And o usuário de CPF "00000000001" pode continuar e concluir seu pedido normalmente
+
+Scenario: Carrinho de compras para finalização de pedido
+Given que o usuário com CPF "00000000001" está na página "principal"
+And o usuário de CPF "00000000001" adiciona os itens "Camisa Puma - masculino" e "Camisa Adidas - masculino" ao carrinho de compras
+When o usuário de CPF "00000000001" vai para a página "carrinho_de_compras"
+Then o usuário de CPF "00000000001" consegue ver os itens "Camisa Puma - masculino" e "Camisa Adidas - masculino"
+And a opção "Finalizar pedido" é exibida e está disponível para ser selecionada
```

Comment:
Based in the previous comment, this scenarios doesn't exist, it'll be covered when you make the scenario of order finishing.